layers/+distributions/spacemacs-base/funcs.el

```diff
@@ -124,6 +124,16 @@ automatically applied to."
     (while (condition-case nil (windmove-right) (error nil))
       (delete-window))))
 
+(defun spacemacs/maximize-vertically ()
+  "Delete all windows left or right of the current window."
```

Comment:
The doc-string needs to be updated to:
`"Delete all windows above and below the current window."`

The doc-string for the `spacemacs/maximize-horizontally` function, also should use `and` instead of `or`.
Resulting in:
`"Delete all windows to the left and right of the current window."`